doc/making-a-visualizer.md

```diff
@@ -23,7 +23,7 @@ the following data and methods:
 4. `sketch`: a `p5` sketch object already live on the page. Usually, this will
    be set by the module manager when a visualizer is created, so all you need
    to do is provide that `initialize()` sets `this.sketch = sketch` (see the
-   `VisualizerDefault` for an example).
+   `SequenceDefault` for an example).
```

Comment:
why did this go from visualizer to sequence?

Reply:
So what you're seeing there is the result of a search and replace that got messed up because of the naming conflict.

Reply:
Fix incoming...

Reply:
Should be fixed in https://github.com/numberscope/frontscope/pull/242/commits/c51aff124c62bb83090720d3e18cee57c2d736a9.